### DISCUSSION TITLE
请问是否有ipv6改造相关的配置教程

### DISCUSSION THREAD

**Discussion by huwenbiao** [apache/apisix#6365] [Q&A] [answered]
看文档中有说明支持使用 IPv6 格式匹配路由，也从其他地方看到了nginx要带--with-ipv6模块编译。想了解下是否有相关的配置教程

**Comment by tokers** [comment, marked as answer]
@huwenbiao Since nginx/1.11.5, `--with-ipv6` was deprecated, and IPv6 will be configured automatically.

**Reply by huwenbiao** [reply]
thanks!

**Comment by zhantanfeng** [comment]
请问你的etcd配置的地址是ipv6吗?有作用吗？

**Reply by huwenbiao** [reply]
没有在apisix上使用，我们openresty是可以的
